lib/verbalize/failure.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'result'
+
+module Verbalize
+  class Failure < Result
+    extend Gem::Deprecate
+
+    def initialize(error)
+      super(outcome: :error, value: error)
+    end
+
+    def error
```

Comment:
We can defer this for now, I will remove until it becomes more apparent as an actual need one way or the other but I haven't loved using `#value` either for failures.  In almost all cases with `Interactor`, people assign failure messages to an `error` or `errors` key on the context object.